File: program/convert-feet-to-inches/ConvertInchesToFeet.c
#include <stdio.h>
int main() {
   float inch,feet;                               //declating a variable
   printf("Enter value in inches = ");
   scanf("%f",&inch);                             //storring value in a variable
   feet = inch / 12;                             // formula for converting inch into feet
   printf("\nyou get in feet  = %.2f ",feet);
}

File: program/program/convert-inches-to-feet/ConvertInchesToFeet.java
import java.util.*;
import java.io.*;
public class Main
{
	public static void main(String[] args) {
	    System.out.print("Please enter the inch value: ");
	    Scanner sc=new Scanner(System.in);
	    int inches=sc.nextInt();
	    int feet = inches/12;
	   System.out.print(inches+" inches is "+feet+ " feet");

	}
}
